app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayViewHolder.kt

```diff
@@ -136,6 +142,20 @@ abstract class TabsTrayViewHolder(
         }
     }
 
+    fun getItemDetails() = object : ItemDetailsLookup.ItemDetails<Long>() {
+        override fun getPosition(): Int = bindingAdapterPosition
+        override fun getSelectionKey(): Long = itemId
+        override fun inSelectionHotspot(e: MotionEvent): Boolean {
+            return browserTrayInteractor?.isMultiSelectMode() == true
+        }
+    }
+
+    fun showTabIsMultiSelectEnabled(isSelected: Boolean) {
+        itemView.selected_mask.isVisible = isSelected
+//        itemView.mozac_browser_tabstray_close.isVisible =
```

Comment:
nit: Is there any TODO related to this?

Reply:
Ah, good catch. That should be linked to https://github.com/mozilla-mobile/fenix/issues/18656